Prototope/Image.swift

```diff
@@ -34,4 +34,108 @@ public struct Image {
 	init(_ image: UIImage) {
 		uiImage = image
 	}
+    
+    public func toPixels() -> PixelBitmap {
+        return PixelBitmap(image: self)
+    }
 }
+
+public struct PixelBitmap : MutableCollectionType {
+    typealias Index = Int
+    
+    init(image: Image) {
+        width = Int(image.size.width)
+        height = Int(image.size.height)
+        
+        data = UnsafeMutablePointer<Pixel>.alloc(width*height)
+        
+        let colorSpace: CGColorSpace = CGColorSpaceCreateDeviceRGB()
+        let bitmapInfo = CGBitmapInfo(CGImageAlphaInfo.PremultipliedLast.rawValue)
+        let bytesPerRow = width * 4
+        
+        let context = CGBitmapContextCreate(data, width, height, 8, bytesPerRow, colorSpace, bitmapInfo)
```

Comment:
Ah, right you are, @NachoSoto, thanks! 
’tis not the context that is leaking (CF objects are automatically memory-managed), but the data allocated on https://github.com/Khan/Prototope/pull/57#discussion-diff-26448524R50.

I added a [`PixelDataDestroyer` class](https://github.com/saniul/Prototope/commit/f7911f32325991ea54b27433f3e0e32ae63c266a#diff-f02caa907b93175e1f8d8ca0f9e24f08R124) to properly clean up the underlying `PixelBitmap` data on deinit.